prow/plugins/plugins.go

```diff
@@ -439,6 +440,14 @@ type MergeWarning struct {
 	BranchWhiteList map[string][]string `json:"branch_whitelist,omitempty"`
 }
 
+// Welcome is config for the welcome plugin
+type Welcome struct {
+	// MessageTemplate is the welcome message template to post on new-contributor PRs
+	// For the info struct see prow/plugins/welcome/welcome.go's PRInfo
+	// TODO(bentheelder): make this be configurable per-repo?
```

Comment:
This would be amazing

Reply:
see https://github.com/kubernetes/test-infra/pull/8643#discussion_r201756881
We can do this for now by eg `{{if eq .Repo "test-infra"}}`, `{{if eq .Org "kubernetes"}}`

Reply:
hahaha, give a man a turing-complete templating language...